### DISCUSSION TITLE
Is there a way to automatically refresh the output of a plugin?

### DISCUSSION THREAD

**Discussion by atlds2005** [swiftbar/SwiftBar#144] [General]
Is there a way to automatically refresh the output of a plugin? I want my plugin to automatically refresh, say, once every hour instead of having me to go and press Cmd+R.

**Comment by melonamin** [comment]
Sure, one can argue that this is the whole point of the app!😁

Two ways of doing that:
* use a special file [name](https://github.com/swiftbar/SwiftBar#plugin-naming)
* or use the [schedule tag](https://github.com/swiftbar/SwiftBar#refresh-schedule) in metadata
